services/github/getLocationByUsername.js

```diff
@@ -9,7 +9,9 @@ export default async function getLocationByUsername(username) {
     lat: 0,
     lon: 0,
   };
-  const octokit = new Octokit({});
+  const octokit = new Octokit({
+    auth: "process.env.GITHUB_API_TOKEN"
```

Comment:
Please make the necessary updates to the .env, .env.example, and .env.docker files by adding the environment variable GITHUB_API_TOKEN. The default value for this variable should be an empty string.

Reply:
In addition to Ravi's comment

- `process.env.GITHUB_API_TOKEN` should not be quoted
- we should not be using `process`, please check the `config/schemas/serverSchema.js`